utils/sharedcache/config.go

```diff
@@ -0,0 +1,32 @@
+package sharedcache
+
+import (
+	"time"
+
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+
+	configUtils "github.com/ARM-software/golang-utils/utils/config"
+)
+
+type Configuration struct {
+	RemoteStoragePath       string        `mapstructure:"remote_storage_path"` // Path where the cache will be stored.
+	Timeout                 time.Duration `mapstructure:"Timeout"`             // Cache timeout if need be
```

Comment:
should be lowercase in the mapstructure?

Reply:
good spot